src/auditlog/compat.py

```diff
@@ -6,10 +6,10 @@ def is_authenticated(user):
 
     Function provides compatibility following deprecation of method call to
     is_authenticated() in Django 2.0.
```

Comment:
As per your comment, maybe add info here that this is only required for Django 1.10 and older.

Reply:
Ok. Added detail to both comment and in-line in the code as well. 

Note that this is only needed for 1.9 and earlier (django 1.10 accepts `is_authenticated` as a property ([1.10 release notes](https://docs.djangoproject.com/en/2.0/releases/1.10/#using-user-is-authenticated-and-user-is-anonymous-as-methods))).